RELEASE.md

```diff
@@ -1,78 +1,100 @@
-# Releasing Quark
+# Releasing Quark #
 
-## Before releasing...
+The release workflow has the following steps, which are scripted.  All
+commands below take additional options, see `./release --help` for
+details.  Publishing artefacts to pypi requires write permission to
+pypi.
 
-Make sure the goals of the release are met. As an example, consider the goals for release 0.1.0.
+## Release script ##
 
-> For a prospective user trying out Quark, we should make sure that
->
-> - they are pointed to the most interesting example we have (slack.q)
-> - prerequisites and install instructions are fully debugged
-> - the example itself works
-> - we have removed any non-working and/or distracting/cluttered examples
-> - the documentation we do provide is fully in sync with the code
->
-> The overall idea is to make sure the process of a) understanding class of problems quark solves, b) trying out the examples we provide, and c) adapting our examples to fit their own specific problems is as streamlined as possible for prospective users.
+Since the release script is part of the repository, and release
+involves switching branches, the release script may not be stable
+across the branches. The best approach is to use two clones of the
+repo, one to provide the script and one to be manipulated by the
+script.
 
-## Prerequisites for every release:
+    # release = $(./release freeze)
```

Comment:
Are you trying to use the "#" character to denote a shell command, or as markup? It's not rendering as markup, and it's confusing to see it used on the same line as "$" later (which is more typically used to denote shell commands).